docs/docs/about/why-cli.mdx

```diff
@@ -2,8 +2,47 @@
 sidebar_position: 1
 ---
 
-# Why this CLI
+# Why this CLI?
 
 CLI for Microsoft 365 is a cross-platform CLI that allows users on any platform to manage various configuration settings of Microsoft 365.
 
-While building solutions for Microsoft 365 expands beyond the Windows operating system, managing many of the platform settings is possible only through PowerShell on Windows. As more and more users work on non-Windows machines, it's inconvenient for them to have to use a Windows virtual machine to configure their tenants. The CLI for Microsoft 365 allows them to configure their tenants no matter which operating system they use.
+- CLI for Microsoft 365 offers a variety of commands from many different areas of Microsoft 365 like: [SharePoint Online](../cmd/spo/spo-get), [Teams](../cmd/teams/app/app-install), [Viva engage](../cmd/viva/engage/engage-search), [Entra](../cmd/entra/administrativeunit/administrativeunit-add), [Power Apps](../cmd/pa/app/app-export), [Power Automate](../cmd/flow/flow-disable), [Power Platform](../cmd/pp/aibuildermodel/aibuildermodel-get), [OneDrive](../cmd/onedrive/onedrive-list), [Planner](../cmd/planner/bucket/bucket-add), [OneNote](../cmd/onenote/notebook/notebook-add), [Outlook](../cmd/outlook/mail/mail-send), [SharePoint Embedded](../cmd/spe/containertype/containertype-add) and many more.
```

Comment:
Let's make this into a list to make it easier to read. Next to naming all these areas, let's emphasize what's the benefit of combining all these areas into one tool, for example:
- consistency (manage all areas in the same way, rather than having to learn how each tool for each area work)
- interoperability (login once, manage everything, easily pass values from one command in one area to another in another area)
- convenience (install, manage, and learn one tool)